docs/source/models.rst

```diff
@@ -358,19 +395,26 @@ platform for real-time serving.
 
     scoring_uri=$(az ml service show --name <deployment-name> -v | jq -r ".scoringUri")
 
+    # `sample_input` is a JSON-serialized Pandas DatFrame with the `split` orientation. The optional
+    # `index` field, which provides labels for DataFrame rows, is not present in this example.
```

Comment:
I would probably remove the mention of index here, and above. A user reading this code is extremely unlikely to find this sentence useful.